package.json

```diff
@@ -344,6 +344,7 @@
     "husky": "^8.0.0",
     "ini": "^2.0.0",
     "mock-fs": "^5.1.2",
+    "netlify-cli": "^12.0.11",
```

Comment:
I don't think we should have the CLI as dependency in the CLI :D

Reply:
oof :D

Reply:
Good catch :my-bad: